src/routes/Tool/Tool.tsx

```diff
@@ -162,15 +162,15 @@ const Tool = () => {
 
   const Desktop = () => {
     return (
-      <div className="flex justify-between">
-        <ul className="m-0 space-x-[16px] p-0">
+      <div className="-mx-5 flex justify-between">
```

Comment:
Using negative margins seemed suitable in this case for the toolbar. Without this there is not enough space for buttons as the content area has padding/margin. 
```
// Mobile breakpoint 640px
640px  / 5 buttons = 128px per one button to use.
```

These spacings on UI-side are no longer needed as the "air" comes already in the DS-side.
And `m-0` were no needed, they seems to be already set to zero.